keras_cv/models/adain.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
lets move adain into it's own directory.

Lets do:

keras_Cv/models/generative/adain/